src/main/java/com/larscheidschmitzhermes/keycloak/events/MonitoringEventListenerProviderFactory.java

```diff
@@ -17,7 +18,13 @@ public EventListenerProvider create(KeycloakSession session) {
 
     @Override
     public void init(Config.Scope config) {
-        this.eventsDirectory = config.get("eventsDirectory");
+        String eventsDirectoryEnvVal = System.getenv(EVENTS_DIRECTORY_ENV);
```

Comment:
can you move that inside the if block? so it's really only ever called as a fallback
also, maybe some debug logging might be nice here to help people figure out what they are doing